src/client.js

```diff
@@ -721,6 +721,15 @@ client.prototype.handleMessage = function handleMessage(message) {
               //
               break;
 
+            case 'msg_rejected':
+              this.log.info(`[${channel}] ${msg}`);
+              this.emit('automodinreview', channel);
+              break;
+            case 'msg_rejected_mandatory':
+              this.log.info(`[${channel}] ${msg}`);
+              this.emit('mandatorymsg', channel);
```

Comment:
Instead of `mandatorymsg`, would something like `automodrejected` be more aligned with `automodinreview` event?

Also, emit `{ channel }` like above.